.github/workflows/tidy-dependencies.yml

```
@@ -18,7 +18,7 @@ jobs:
           token: ${{ secrets.OPENTELEMETRYBOT_GITHUB_TOKEN }}
       - uses: actions/setup-go@v4
         with:
-          go-version: 1.19
+          go-version: 1.20
```

```suggestion
          go-version: ~1.20.7
```
This would align this setup go with other worfklows.